src/cpp/server/server_cc.cc

```diff
@@ -417,7 +417,9 @@ class Server::SyncRequest final : public grpc::internal::CompletionQueueTag {
                                  : server_->resource_exhausted_handler_.get();
       deserialized_request_ = handler->Deserialize(call_, request_payload_,
                                                    &request_status_, nullptr);
-
+      if (!request_status_.ok()) {
+        gpr_log(GPR_ERROR, "Failed to deserialize message.");
```

Comment:
Put another way: this is probably a GPR_ERROR for internal use where we know both the client and the server but I would be loath to call it such in the Internet case where anything can be sending to anywhere.

Reply:
I could see this as either `INFO` or `DEBUG`. I guess looking at comparative use, it should probably be `DEBUG` since we do use that for things like missing metadata, etc.